tests/test_pipelines/test_rasterize.py

```diff
@@ -71,49 +63,51 @@ def add_vector_data(pipeline):
         eopatch.save(eopatch_folder, features=[(FeatureType.VECTOR_TIMELESS, "LULC_VECTOR")], overwrite_permission=1)
 
 
+def reset_output_folder(config_path: str) -> Optional[str]:
```

Comment:
Had to write this utility function to reset the output folder before running the pipelines. I feel this is a worthwhile addition, since the rest of the file then looks quite nice IMO (at least compared to before)